### PR TITLE
perf: add `skipRuntimeLoading` option to skip generating runtime code

### PR DESCRIPTION
**Pull request by aleen42**
According to #256, we need an option to skip generating runtime code for loading asynchronous chunks.

### CODE REVIEW

**Comment by aleen42** [comment]
@alexander-akait It seems that `prettier` has a bug? And I can't see any error information in the pipeline.

File: package.json [hunk 1]
```diff
@@ -57,7 +57,6 @@
     "cross-env": "^7.0.3",
     "css-loader": "^5.2.6",
     "del": "^6.0.0",
-    "del-cli": "^4.0.0",
```
Comment (alexander-akait) [inline package.json:60]: Don't touch this

Comment (aleen42) [reply package.json:60]: The library has broken under Windows.

**Comment by alexander-akait** [comment]
https://github.com/webpack-contrib/mini-css-extract-plugin/pull/832, please try it, feel free to feedback, sorry for delay

**Comment by aleen42** [comment]
@alexander-akait good job

**Pull request closed without merging**